src/Rasa.Game/Data/EntityGUID.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+
+namespace Rasa.Data
+{
+    public struct EntityGUID : IEquatable<EntityGUID>
+    {
+        public ulong Id { get; private set; }
+        public EntityType EntityType { get; private set; }
```

Comment:
Please remove this property and implement it like we've talked about in Discord. Thanks!